src/util/ChartUtils.ts

```diff
@@ -247,10 +249,12 @@ export const getBarSizeList = ({
           result[cateId] = [];
         }
 
+        const barSize: string | number | undefined = isNil(selfSize) ? globalSize : selfSize;
```

Comment:
I wonder if we should use `invariant` and throw to the user an error message if their input is a string but not a percentage? 

None of the other recharts code does that I don't think for the items that accept percentages, but we probably should. All good for now